docs/installation/Installation.md

```diff
@@ -83,7 +83,7 @@ Because we use [HOCON](../installation_configuration_guide/Common.md#conventions
 | HTTP_INTERFACE                | string          | 0.0.0.0                                                                                                                                       | Network address Nussknacker binds to                                                                                                                                                                                                                      |
 | HTTP_PORT                     | string          | 8080                                                                                                                                          | HTTP port used by Nussknacker                                                                                                                                                                                                                             |
 | HTTP_PUBLIC_PATH              | string          |                                                                                                                                               | Public HTTP path prefix the Designer UI is served at, e.g. using external proxy like [nginx](#configuring-the-designer-with-nginx-http-public-path)                                                                                                       |
-| DB_URL                        | string          | jdbc:hsqldb:file:${STORAGE_DIR}/db;sql.syntax_ora=true                                                                                        | Database URL                                                                                                                                                                                                                                              |
+| DB_URL                        | string          | jdbc:hsqldb:file:${STORAGE_DIR}/db;sql.syntax_ora=true                                                                                        | Database; see [this page](../installation_configuration_guide/DesignerConfiguration.md#database-configuration) for more details                                                                                                                           |
```

Comment:
I can't see any more details there. What do you exactly mean by that?

Reply:
Changed it to "See also". 

I added this link because we have several databases mentioned in the docs (Influx, enrichers, this one) and it is not obvious which one is meant here. Additionally, the db in question does not stand out in our docs (why should it, anyway?), so making the connection is difficult.